apps/extension/src/App/App.tsx

```diff
@@ -1,14 +1,37 @@
 import { Container } from "@namada/components";
 import { useVaultContext } from "context/VaultContext";
+import { useRequester } from "hooks/useRequester";
+import { CheckDurabilityMsg } from "provider";
+import { useEffect, useState } from "react";
 import { matchPath, useLocation } from "react-router-dom";
+import { Ports } from "router";
 import { AppContent } from "./AppContent";
 import { AppHeader } from "./Common/AppHeader";
 import { Login } from "./Login";
 import routes from "./routes";
 
+const STORE_DURABILITY_INFO =
+  'Store is not durable. This might cause problems when persisting data on disk.\
+ To fix this issue, please navigate to "about:config" and set "dom.indexedDB.experimental" to true.';
+
 export const App: React.FC = () => {
   const location = useLocation();
 
+  const [warnings, setWarnings] = useState<string[]>([]);
```

Comment:
I think we could leave it here for now, but maybe in the future it would be good if we move this to the store, so other parts of the extension could add a warning to the list (but let's not make this premature refactor :P)

Reply:
I think that makes sense! I was thinking it would be a good idea to approach all of the apps in a similar way, we could just use a Jotai store here. I definitely didn't want to start a refactor for this release